src/lib/helpers/notifications.ts

```diff
@@ -19,12 +19,12 @@ const userPreferences = () => get(user)?.prefs;
 const notificationPrefs = (): Record<string, NotificationPrefItem> => {
     const prefs = userPreferences();
 
-    if (prefs === null) {
+    // due to php backend, empty object can be returnd as an empty array
+    if (prefs?.notificationPrefs === null || Array.isArray(prefs?.notificationPrefs)) {
```

Comment:
what if `prefs` is `undefined`?

Reply:
also fixed 👍🏻 improved the condition handling